lib/telnet.c

```diff
@@ -81,8 +81,6 @@
   } WHILE_FALSE
 
 #define  CURL_SB_GET(x) ((*x->subpointer++)&0xff)
-#define  CURL_SB_PEEK(x)   ((*x->subpointer)&0xff)
-#define  CURL_SB_EOF(x) (x->subpointer >= x->subend)
```

Comment:
I think these have valuable information even if we're not using them right now, so I'd leave them alone. If Daniel wants that warning fixed I would comment them out like `/* For posterity:` or something. The rest look ok to jettison.